resources/authgear/templates/en/web/authflowv2/settings_profile_edit_birthdate.html

```diff
@@ -0,0 +1,40 @@
+{{ template "authflowv2/__html_head.html" . }}
+{{ template "authflowv2/__settings_page_frame.html" . }}
+
+{{ define "page-content" }}
+
+{{ template "authflowv2/__navbar.html"
+(dict
+  "BackHref" (call $.MakeURL "/settings/profile")
```

Comment:
Thinking if BackTitle (The word "Back") should be included for desktop view @cychiuae